release/reOscar.sh

```
@@ -0,0 +1,112 @@
+
+# --- sanity check run as root
+if [ "$(id -u)" != "0" ];
+then
+        echo `basename "$0"` "script must be run as root" 1>&2
+        exit 1
+fi
+
+
+# --- select the running Tomcat or the highest installed version
+TOMCAT=$(ps aux | grep org.apache.catalina.startup.Bootstrap | grep -v grep | awk '{ print $1 }')
+TOMCAT_PID=$(ps aux | grep org.apache.catalina.startup.Bootstrap | grep -v grep | awk '{ print $2 }')
```

Shellcheck (0.11.0)

[warning] TOMCAT_PID appears unused. Verify use (or export if used externally).

(SC2034)